doc/classes/AnimationNodeStateMachine.xml

```diff
@@ -3,6 +3,13 @@
 	<brief_description>
 	</brief_description>
 	<description>
+		Contains multiple root nodes as children in a graph.
+		Each node is used as a state, and provides multiple functions to alternate between states.
+		Retrieve the AnimationNodeStateMachinePlayback object from the [AnimationTree] node to control it programatically.
+		[codeblock]
+			var state_machine = anim_tree["parameters/StateMachine/playback"]
```

Comment:
Code blocks don't need to be further indented. If you do need indentation (e.g. for a `func:` or `if:` block, use 4 spaces).